README.md

```diff
@@ -60,6 +60,291 @@ environment variables manually (the actual bus address might be different):
 	process.env.DISPLAY = ':0';
 	process.env.DBUS_SESSION_BUS_ADDRESS = 'unix:path=/run/dbus/system_bus_socket';
 
+
+## API
+
+
+### DBus
+
+The root object of this module.
+
+#### `new DBus()`
+
+Create a new DBus instance.
+
+```
+var DBus = require('dbus')
+var dbus = new DBus()
+```
+
+#### `DBus.prototype.getBus(busName)`
+
+* busName `<string>`
+
+Connect to a bus. `busName` must be either `"system"` to connect to the system
+bus or `"session"` to connect to the session bus.
+
+Returns a `Bus`.
+
+```
+var bus = dbus.getBus('session');
+```
+
+#### DBus.prototype.registerService(busName, serviceName)
```

Comment:
also wrap it with backtick:  `DBus.prototype.registerService(busName, serviceName)`